loop_rate:      1.5   # The frequency in Hz at which to run the control loops.
queue_size:     1     # The size of the message queue used for publishing and subscribing to topics.
swarm_timeout:  5.0   # Time in seconds after which it is assumed that a swarm member has left the swarm if no position update has been received.
visualize:      true  # Whether to publish the coverage path on a topic for visualization.
